client/src/TenderList.css

```diff
@@ -75,6 +75,82 @@ td[data-status="Closed"] {
 	color: #8a226f;
 }
 
+.tender-id {
+	font-weight: 700;
+}
+
+.tender-id:hover {
+	color: #ec221f;
+	text-decoration: underline;
+}
+
+/*======New UI for Tables=====*/
```

Comment:
The title of this PR is "Buyer accept or Reject bid", the new UI for tables shouldn't be in this PR

Reply:
I have only updated the ui For the userstory 5.2.6